scripts/tablegen/tablegen.py

```diff
@@ -132,7 +136,13 @@ def decode_helm_yaml(s):
         print d
         if "<!-- AUTO-GENERATED-START -->" in d:
             break
+    # if os.path.exists("values.yaml"):
+    #     os.remove("values.yaml")
+    # else:
+    #     print("values.yaml doesn't exist")
 
+    # values = 'https://raw.githubusercontent.com/istio/istio/master/install/kubernetes/helm/istio/values.yaml'
+    # values_yaml = wget.download(values)
```

Comment:
This will be a little challenging as the subcharts also need their values.yaml read.  The current code sorts the outputs by service name.  One option is to read the requirements.yaml (via wget) in the top level chart, and add any values.yaml found in the subcharts (via wget) to the yaml that is retrieved from the branch or tag.